### PR TITLE
Add missing blog URLs to bugs.json

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: docs/bugs.json [hunk 1]
```diff
@@ -73,6 +74,7 @@
         "name": "SignedArrayStorageCopy",
         "summary": "Assigning an array of signed integers to a storage array of different type can lead to data corruption in that array.",
         "description": "In two's complement, negative integers have their higher order bits set. In order to fit into a shared storage slot, these have to be set to zero. When a conversion is done at the same time, the bits to set to zero were incorrectly determined from the source and not the target type. This means that such copy operations can lead to incorrect values being stored.",
+        "link": "https://blog.ethereum.org/2019/06/25/solidity-storage-array-bugs/",
```
Comment (axic) [inline docs/bugs.json:77]: Should we change all these URLs to `solidity.ethereum.org` ?

**Comment by axic** [comment]
Also noticed https://blog.ethereum.org/2016/11/01/security-alert-solidity-variables-can-overwritten-storage/ was followed by an update https://blog.ethereum.org/2016/11/09/analysis-storage-corruption-bug/, but we don't have that listed (since there's a single "link" field).

Also mentioned this in https://github.com/ethereum/solidity-blog/issues/37

File: docs/bugs.json (hunk 1 above)
Comment (leonardoalt) [reply docs/bugs.json:77]: If they work, why not?

Comment (axic) [reply docs/bugs.json:77]: cc @franzihei @chriseth

Comment (chriseth) [reply docs/bugs.json:77]: I wonder if the URLs will stay stable, but we can always change it again in that list.

Concerning stability of the URLs: Maybe the blog posts should be under `solidity.ethereum.org/blog/date/post`

**Review by chriseth — APPROVED** [review]

**Pull request merged**